pkg/transparentproxy/config/config.go

```diff
@@ -140,6 +151,26 @@ type Config struct {
 	// Log is the place where configuration for logging iptables rules will
 	// be placed
 	Log LogConfig
+	// RestoreLegacy indicates whether the ip{6}tables-restore version is legacy
+	// (non-nftables). If RestoreLegacy is true, the flags "--wait" and
+	// "--wait-interval" are valid. These flags are invalid with nftables.
```

Comment:
Do we need this flag at all? Can't we detect if it's xtables vs nftables?

might be worth mentioning that this is not needed with nftables

```
With back end transactions being atomic, there is no need for the global xtables lock which has proven problematic in environments with large and/or rapidly changing rulesets
```

Reply:
It was not a flag, but just a Config property, as I was checking this in earlier stage of the installation, but I moved it now and got rid of the property completely